plugins/modules/zos_operator_action_query.py

```diff
@@ -395,34 +396,34 @@ def filter_requests(merged_list, params):
     message_id = params.get("message_id")
     job_name = params.get("job_name")
     newlist = merged_list
-
     if system:
         newlist = handle_conditions(newlist, "system", system)
     if job_name:
         newlist = handle_conditions(newlist, "job_name", job_name)
     if message_id:
         newlist = handle_conditions(newlist, "message_id", message_id)
-
     return newlist
 
 
 def handle_conditions(list, condition_type, value):
```

Comment:
Could we change this function a bit and use different variable names for `list` and `dict`? We could have problems since these may shadow other Python objects